bookomate/templates/base.html

```diff
@@ -11,6 +11,13 @@
         <link rel="stylesheet" href="{% static 'css/style.css' %}">
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
         <script src="https://kit.fontawesome.com/189832a678.js" crossorigin="anonymous"></script>
+<<<<<<< HEAD

```

Comment:
Please remove `git diff` indicators.

Reply:
Sorry, this is a super noob question to ask, but how do I approve your requested changes?

Reply:
You cannot now because I didn't make changes. Just remove these indicators on your local copy and push changes :)

Reply:
Ok, the changes have been pushed!